### ISSUE TITLE
Performance issue on the IAS server

### ISSUE THREAD

**Issue by helioshk** [Helioviewer-Project/api#420]
Since a few days the images take very long to load. 
About 10 minutes for the 80 frames, accessing with JHV. It would be good if someone could have a look.
Thanks.

**Comment by helioshk** [comment]
bump

**Comment by dgarciabriseno** [comment]
@ebuchlin

**Comment by ebuchlin** [comment]
On my side it is about 2min for 80 AIA frames, with a fast connection from out of the lab, this is less slow but still too slow. We don't understand why at the moment. We also had some issues in the last days about the database being OOM-killed (while we normally have enough memory), maybe this is related, maybe not.

**Comment by helioshk** [comment]
8 minutes 10 seconds for 23 GOES SUVI frames.

**Comment by helioshk** [comment]
At the moment it appears to be fast, even faster than before the problems were showing up. I'm getting about 80 Mbit/s through, before it was only 40 Mbit/s.

**Comment by helioshk** [comment]
Issue appears to be solved.

**Comment by ebuchlin** [comment]
Indeed, Université Paris-Saclay IT has replaced some IT equipment, things look better now.
